contrib/cirrus/test.sh

```diff
@@ -23,17 +23,12 @@ then
         validate)
             showrun ooe.sh git remote add upstream "$CIRRUS_REPO_CLONE_URL"
             showrun ooe.sh git remote update
-            if [[ -z "$CIRRUS_PR" ]]
-            then
-                echo "Testing a branch, assumed or based on the $DEST_BRANCH branch from .cirrus.yml"
-                export GITVALIDATE_EPOCH="$(git rev-parse upstream/$DEST_BRANCH)"
-            else  # Testing a PR
-                echo "Testing a PR targeted at the $DEST_BRANCH branch"
-                export GITVALIDATE_EPOCH="$(git merge-base upstream/$DEST_BRANCH HEAD)"
+            if [[ -n "$CIRRUS_PR" ]]; then
+                export GITVALIDATE_EPOCH="$CIRRUS_CHANGE_IN_REPO"
+            else
+                export GITVALIDATE_EPOCH="$CIRRUS_CIRRUS_LAST_GREEN_CHANGE"
             fi
-            export GITVALIDATE_TIP="$CIRRUS_CHANGE_IN_REPO"
```

Comment:
Is this tip no longer needed?

Reply:
It's a good question.  I think it's not necessary since [the script defaults to HEAD] when unset(https://github.com/containers/buildah/pull/2618/files#diff-4981a77767bb9bd180e550926cdb2df4R25).  Thinking more though, I think `$GITVALIDATE_EPOCH` should actually be `$CIRRUS_BASE_SHA`.  Hmm.  [The docs are really not very clear about either of these.  ](https://cirrus-ci.org/guide/writing-tasks/#environment-variables) :confused: